package-lock.json

```diff
@@ -160,6 +160,15 @@
             "uri-js": "^4.2.2"
           }
         },
+        "debug": {
```

Comment:
@1cm can you remove the package-lock update from this PR please? Thanks

Reply:
@samvloeberghs there is only delete file option here https://github.com/samvloeberghs/kwerri-oss/pull/40/commits/3a2f4dc05eaa0f78d552a1216f9748d7346c087a
Should i delete the file here?